### ISSUE TITLE
Motion detection triggered when switching between day- and nightmode

### ISSUE THREAD

**Issue by Trekky12** [EliasKotlyar/Xiaomi-Dafang-Hacks#883]
I really like the effort of all of you for making this hack possible and I really like it very much.

I'm using the motion detection and I noticed that a switch between day- and nightmode is also triggering a motion detection. Is there anything that can prevent this? Does anybody has the same problem?

The camera used is a Xiaomi Dafang on commit caff4f76a1804ece2d64b985ee2334baeaf2d263

**Comment by lenr9** [comment]
I am having the same issue.  Motion sent emails for nearly 3 hours tonight as the sun was setting and different amount of light was coming through the windows of the room with the camera.  5 pics per email and you can see some in color and some black and white.  Firmware version 2019-01-31T14:59:50Z 

![image](https://user-images.githubusercontent.com/47092471/52159623-bad65980-265b-11e9-9d91-1ffa9bf92b43.png)
![image](https://user-images.githubusercontent.com/47092471/52159627-c32e9480-265b-11e9-9086-7c5ff5e9a9d6.png)

**Comment by lenr9** [comment]
Today I changed my settings under Configure Motion Settings to Moderate and did not experience the flood of emails like yesterday.  Tested motion by walking around in the room and did get emails for that motion.  It was raining all day today so I hope the lack of full sunlight did not impact the results of this testing.

Do not have a camera pointing outdoors yet, but will test that out next week when I get one installed.  For now, for indoor cameras, this appears to be the solution.
